.travis/pre_before_install.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+pip install -r test_requirements.txt
+./plugin-template --generate-config --plugin-app-label catdog pulp_catdog
+mkdir ../pulp_catdog/.travis
+touch ../pulp_catdog/.travis/test_bindings.py
+echo 'pypi_username: the_pypi_user' >> ../pulp_catdog/template_config.yml
+./plugin-template --all pulp_catdog
+mv ../pulp_catdog/*.txt .
+mv ../pulp_catdog/.travis/before_install.sh ../pulp_catdog/.travis/old_before_install.sh
```

Comment:
I had to make some changes on `before_install.sh`, so I'm not using the generated `before_install.sh`.
This is the change that most worries me, suggestions are very welcome!